app/views/carts/_cart.html.erb

```diff
@@ -9,10 +9,12 @@
     <div class="col-md-4" id="item_<%= item.id %>">
       <p><%= item.title %></p>
       <p><%= item.description %></p>
-      <p><%= quantity %> x <%= number_to_currency(item.price_in_dollars) %></p>
-      <%= link_to "Remove", cart_path(item_id: item.id), method: :delete %>
-    </div>
+      <p><%= quantity %> x <%= number_to_currency(item.price_in_dollars) %>
+      <%= link_to "-", cart_path(item_id: item.id), method: :delete %>
+      <%= link_to "+", cart_path(item_id: item.id), method: :post%></p>
+    <%= link_to "Remove", cart_path(item_id: item.id, :remove => "remove"), method: :delete %>
   </div>
```

Comment:
This seems like a standard way to handle "+" and "-".

During our interface session today, lets make these into custom buttons. They will probably look very similar, but just bigger and more obvious that they can be clicked.